notebooks/samples/Vowpal Wabbit Overview.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
Nit to keep with the style of others lets make title Vowpal Wabbit - Overview. Likewise for other NBs